### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HKAgrwl**
Added tech stack icons and Discord link+icon

### CODE REVIEW

**Comment by rohitdasu** [comment]
hi @HKAgrwl could you please make sure images and text aligned? seems like images are placed little down. see the image below 
Screenshot 2022-08-22 at 8 15 50 PM

**Comment by HKAgrwl** [comment]
Yeah, I'll fix it...

**Comment by rohitdasu** [comment]
> Yeah, I'll fix it...

thanks @HKAgrwl

**Comment by HKAgrwl** [comment]
I cant't use CSS or html style in readme.md as github doesn't allow that. Still made some improvements. Please review....

File: README.md [hunk 1]
```diff
@@ -2,10 +2,13 @@
 A web app where you can find contributors for your open-source project or as an individual contributor you can find open-source projects. Best place to start building in public.
 Find people who have the same vision as you do and contribute to each other for building amazing products.
 
+## Connect with us
+<p align='left'>Discord:  &nbsp &nbsp <a href="https://discord.gg/M2BMPdku" target="_blank" rel="noreferrer"> <img src="https://www.svgrepo.com/show/353655/discord-icon.svg" alt="Discord" width="30" height="30" align="top"/> </a></p>
+
```
Comment (rohitdasu) [inline README.md:7]: let's remove the discord text, only keep icon as we are facing alignment issue

**Review by rohitdasu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HKAgrwl) [reply README.md:7]: cool

**Comment by rohitdasu** [comment]
hi @HKAgrwl are you going to work on this or should i pick this up? please let me know

**Comment by HKAgrwl** [comment]
I have already raised a PR.Please look into it and suggest any further changes ...

**Comment by rohitdasu** [comment]
@HKAgrwl you haven't worked on the request changes yet. please look into it.

**Pull request closed without merging**